<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConditionWidget</class>
 <widget class="QWidget" name="ConditionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>175</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>175</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="margin">
    <number>3</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="margin">
       <number>3</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>If</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="margin">
          <number>3</number>
         </property>
         <item>
          <widget class="QComboBox" name="m_expression">
           <property name="minimumContentsLength">
            <number>2</number>
           </property>
           <item>
            <property name="text">
             <string>&lt;</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>&lt;=</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>=</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>&gt;=</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>&gt;</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_value"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Then</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Level</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="m_level">
             <property name="minimumContentsLength">
              <number>7</number>
             </property>
             <item>
              <property name="text">
               <string>None</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Critical</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Error</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Warning</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Debug</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Info</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Message</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPlainTextEdit" name="m_message">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>40</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
